modules/serverless-create-eventing-namespace.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-eventing.adoc
+
+[id="serverless-create-eventing-namespace_context"]
+= Creating the `knative-eventing` namespace
+
+When you create the `knative-eventing` namespace, a `knative-eventing` project will also be created.
+
+[IMPORTANT]
+====
+You must complete this procedure before installing Knative Eventing.
+====
+
+If the `KnativeEventing` object created during Knative Eventing's installation is not created in the `knative-eventing` namespace, it will be ignored.
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
+* Installed {ServerlessOperatorName}.
+
+[id="serverless-create-eventing-project-web-console_{context}"]
+== Creating the `knative-eventing` namespace using the web console
+
+.Procedure
+. In the {product-title} web console, navigate to *Administration* → *Namespaces*.
+. Click *Create Namespace*.
++
+image::serverless-create-namespaces.png[Namespaces page]
+. Enter `knative-eventing` as the *Name* for the project. The other fields are optional.
++
+image::create-eventing-namespace.png[Creating the `knative-eventing` namespace]
+. Click *Create*.
+
+[id="serverless-create-eventing-project-oc_{context}"]
+== Creating the `knative-eventing` namespace using the CLI
+
+.Procedure
+. Create the `knative-eventing` namespace by creating the `knative-eventing` project, by entering:
```

Comment:
I remember having a conversation about IBM guidelines on this with Jason. AFAIK we drop "the following command" and just use enter or type, so that's why.

Updated this step anyway since it was redundant after moving these into their own sections.